File: 이훈일/삼성 SW 역량 테스트 기출 문제/13229.java
import java.util.Scanner;
import java.io.FileInputStream;
import java.util.*;

class Solution {
	public static void main(String args[]) throws Exception {
		Scanner sc = new Scanner(System.in);
		int T;
		T=sc.nextInt();
		 
		for(int test_case = 1; test_case <= T; test_case++) {
			List<String> weekend = new ArrayList<String>(){{
            	add("MON");
                add("TUE");
                add("WED");
                add("THU");
                add("FRI");
                add("SAT");
                add("SUN");
            }};
            String today = sc.next();
            int answer = 0;
            if(today.equals("SUN")) {
            	answer = 7;
            }else {
            	answer = 7 - weekend.indexOf(today) - 1;
            }
            System.out.println("#" + test_case + " " + answer);
		}
	}
}

File: 이건희/SWexpertacademy/D3/13229.py
T = int(input())
Day = {"MON" : 6, "TUE" : 5, "WED" : 4, "THU" : 3, "FRI" : 2, "SAT" : 1, "SUN" : 7}
for test_case in range(1, T + 1):
    day = input()
    print(f'#{test_case} {Day[day]}')
